packages/auto-label/src/extensions.json

```diff
@@ -0,0 +1,178 @@
+{
```

Comment:
Do you forsee much updating to this file? Just thinking ahead, every time we've had these files they've become a pain to upkeep. Maybe there's a place you can grab this info from is dynamic?

Reply:
Good point! I do plan on updating this file, and I wasn't able to find an api/resource that dynamically provides file extension to language mappings. 

One of the items on my future todo is to improve this mapping to the point where it can be a canonical set of extension<>language mappings that users will be happy with as a default.

Pls let me know if you know if of a better mapping source. Open to suggestions!